Components/Services/PaymentService.php

```diff
@@ -478,16 +476,17 @@ private function prepareRequest(
             'method' => $paymentMethod,
         ];
 
-        $paymentDescription = (string) (time() . $transaction->getId() . substr($transaction->getBasketSignature(), -4));
+        $paymentDescription = (string)(time() . $transaction->getId() . substr($transaction->getBasketSignature(), -4));
```

Comment:
-4 as constant?

Reply:
haha, i can see what the hell it even is :D

Reply:
ok i have no clue what -4 is , and dont feel good in changing it in this PR :)